TombEngine/Game/Lara/lara.h

```diff
@@ -53,7 +53,7 @@ constexpr auto LARA_RADIUS_UNDERWATER = 300;
 constexpr auto LARA_RADIUS_DEATH	  = 400;
 constexpr auto LARA_ALIGN_VELOCITY	  = 12; // TODO: Float.
 
-constexpr auto LARA_FREEFALL_VELOCITY   = 131.0f;
+constexpr auto LARA_FREEFALL_VELOCITY	= 126.0f;
```

Comment:
This is a fundamental constant. It can't be changed to fix this issue.

Reply:
Any idea of how to solve that? 
![emoticonPonder_64](https://github.com/MontyTRC89/TombEngine/assets/60930991/c0ee5617-30a4-49f6-ad15-52767a2f8174)

Reply:
Yes, it's related to state functions. I have a pending solution but it's not much of a priority.